AWSComprehend/AWSComprehendModel.h

```diff
@@ -3344,52 +4014,305 @@ typedef NS_ENUM(NSInteger, AWSComprehendTargetedSentimentEntityType) {
+ */
+@property (nonatomic, strong) NSString * _Nullable flywheelArn;
+
+/**
+ <p>Last modified time for the flywheel.</p>
+ */
+@property (nonatomic, strong) NSDate * _Nullable lastModifiedTime;
+
+/**
+ <p>The most recent flywheel iteration.</p>
+ */
+@property (nonatomic, strong) NSString * _Nullable latestFlywheelIteration;
+
+/**
+ <p>A description of the status of the flywheel.</p>
+ */
+@property (nonatomic, strong) NSString * _Nullable message;
+
+/**
+ <p>Model type of the flywheel's model.</p>
+ */
+@property (nonatomic, assign) AWSComprehendModelType modelType;
+
+/**
+ <p>The status of the flywheel.</p>
+ */
+@property (nonatomic, assign) AWSComprehendFlywheelStatus status;
+
+/**
+ <p>Configuration about the custom classifier associated with the flywheel.</p>
+ */
+@property (nonatomic, strong) AWSComprehendTaskConfig * _Nullable taskConfig;
+
+@end
+
+/**
+ <p>Flywheel summary information.</p>
+ */
+@interface AWSComprehendFlywheelSummary : AWSModel
+
+
+/**
+ <p>ARN of the active model version for the flywheel.</p>
  */
-@property (nonatomic, assign) AWSComprehendJobStatus jobStatus;
+@property (nonatomic, strong) NSString * _Nullable activeModelArn;
 
 /**
- <p>The language code of the input documents.</p>
+ <p>Creation time of the flywheel.</p>
  */
-@property (nonatomic, assign) AWSComprehendLanguageCode languageCode;
+@property (nonatomic, strong) NSDate * _Nullable creationTime;
 
 /**
- <p>A description of the status of the events detection job.</p>
+ <p>Amazon S3 URI of the data lake location. </p>
  */
-@property (nonatomic, strong) NSString * _Nullable message;
+@property (nonatomic, strong) NSString * _Nullable dataLakeS3Uri;
 
 /**
- <p>The output data configuration that you supplied when you created the events detection job.</p>
+ <p>The Amazon Resource Number (ARN) of the flywheel</p>
  */
-@property (nonatomic, strong) AWSComprehendOutputDataConfig * _Nullable outputDataConfig;
+@property (nonatomic, strong) NSString * _Nullable flywheelArn;
 
 /**
- <p>The time that the events detection job was submitted for processing.</p>
+ <p>Last modified time for the flywheel.</p>
  */
-@property (nonatomic, strong) NSDate * _Nullable submitTime;
+@property (nonatomic, strong) NSDate * _Nullable lastModifiedTime;
 
 /**
- <p>The types of events that are detected by the job.</p>
+ <p>The most recent flywheel iteration.</p>
  */
-@property (nonatomic, strong) NSArray<NSString *> * _Nullable targetEventTypes;
-
-@end
+@property (nonatomic, strong) NSString * _Nullable latestFlywheelIteration;
 
 /**
- <p>Array of the number of characters extracted from each page.</p>
+ <p>A description of the status of the flywheel.</p>
  */
-@interface AWSComprehendExtractedCharactersListItem : AWSModel
-
+@property (nonatomic, strong) NSString * _Nullable message;
 
 /**
- <p>Number of characters extracted from each page.</p>
+ <p>Model type of the flywheel's model.</p>
  */
-@property (nonatomic, strong) NSNumber * _Nullable count;
```

Comment:
Breaking changes

Reply:
This model and its properties still exists: https://github.com/aws-amplify/aws-sdk-ios/blob/03dddc45742b491dbec9238c624a54fd57b0931b/AWSComprehend/AWSComprehendModel.h#L4057